File: Exercicios-Beecrowd/C#/1005.cs
using System;

class Mult{
    static void Main(string[] args){
        double a = Convert.ToDouble(Console.ReadLine());
        double b = Convert.ToDouble(Console.ReadLine());
        a *= 3.5;
        b *=7.5;
        double soma = (a+b)/11;
        string media = soma.ToString("F5");
        
        Console.WriteLine("MEDIA = "+ media);

    }
}

File: Exercicios-Beecrowd/Kotlin/1005.kt
import java.util.*

fun main(args: Array<String>) {

	val valor1= readLine()
    val valor2= readLine()
    if (valor1 != null && valor2!=null){
        val nota1=valor1.toDouble()
        val nota2=valor2.toDouble()
        val resultado= ((nota1*3.5)+(nota2*7.5))/11
        val formatado = ("%.5f".format(resultado))
        println("MEDIA = $formatado")
    }
	
}